src/components/Wrapper.tsx

```diff
@@ -47,82 +52,75 @@ export const Wrapper: preact.FunctionComponent<ContainerProps> = ({
 
   return (
     <div
-      className={
-        'fixed top-0 bottom-0 left-0 right-0 bg-[#14171A33] bg-opacity-20'
-      }
+      className={cn(
+        webPositionClasses,
+        !fullScreen && isMobile ? 'absolute bottom-0' : ''
+      )}
       id={'integraflow-content-wrapper'}
     >
-      <div className='relative w-full h-full'>
-        <div
-          className={cn(
-            webPositionClasses,
-            !fullScreen && isMobile ? 'absolute bottom-0' : ''
-          )}
-        >
+      <div
+        className={cn(
+          'p-6 flex flex-col items-center',
+          !fullScreen && !isMobile ? 'shadow-3xl rounded-2xl w-fit max-h-[600px]' : '',
+          !fullScreen && isMobile
+            ? 'rounded-t-2xl w-screen max-h-[600px] shadow-3xl'
+            : '',
+          fullScreen ? 'w-screen h-screen' : ''
```

Comment:
@jdamilola You missed the fullscreen styles.
```suggestion
          fullScreen ? 'w-screen h-screen fixed z-[2147483650] inset-0' : ''
```